src/features/dashboard/components/ViewTabDropdown/ViewTabDropdown.module.css

```
@@ -0,0 +1,236 @@
+
+.chevron {
+	transition: transform 0.2s ease;
+}
+
+.chevronOpen {
+	transform: rotate(180deg);
+}
+
+/* ── Dropdown panel ─────────────────────────────────────── */
+
+@keyframes dropdownIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "dropdownIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)